project/FrgaalJavaCompiler.scala

```diff
@@ -9,19 +9,20 @@
  * (http://www.apache.org/licenses/LICENSE-2.0).
  */
 
-import sbt._
+import sbt.*
```

Comment:
Looks like an unnecessary _no-semantic change_. Needless to say I prefer non-start - e.g. explicit imports.

Reply:
Reverted in 7df60c5346f11f7baaeb55727243bd1290519065 . Turns out my IDE thinks that Scala sources in the `project` directory are Scala 3 rather than Scala 2.